README.md

```diff
@@ -6,6 +6,21 @@ tree-sitter-bash
 
 Bash grammar for [tree-sitter](https://github.com/tree-sitter/tree-sitter).
 
+### Development
+
+Install the dependencies:
+
+  npm install
```

Comment:
I think that for this code to render well, you have to enclose these shell script lines in triple-backtick code blocks.

Reply:
Thanks. Should be fixed now.